### PR TITLE
Respond non-ascii characters

### PR DESCRIPTION
**Pull request by hamada2029**
Read python source code with non-ascii characters.

### CODE REVIEW

File: bin/pylama/core.py [hunk 1]
```diff
@@ -186,7 +186,7 @@ def __enter__(self):
         """ Open a file and read it. """
         if self.code is None:
             LOGGER.info("File is reading: %s", self.path)
-            self._file = open(self.path, 'rU')
+            self._file = open(self.path, 'rU', encoding='utf8')
```
Comment (gmist) [inline bin/pylama/core.py]: Incompatible with Python 2.x

Comment (gmist) [inline bin/pylama/core.py]:
``` python
import io
...
self._file = io.open(self.path, 'rU', encoding='utf8')
```

Comment (gmist) [inline bin/pylama/core.py]: or

``` python
            self._file = open(self.path, 'rU')
            self.code = self._file.read().decode('utf-8')
```

Comment (hamada2029) [inline bin/pylama/core.py]: Thank you for your advice. You are right.
I forgot py2.

**Pull request closed without merging**